src/node.ts

```diff
@@ -1,7 +1,7 @@
 import * as swc from "@swc/core";
 import fs from "fs";
 import deepClone from "lodash.clonedeep";
-import escapeRegExp from "lodash.escaperegexp";
+import { escapeRegExp, pathPatternToRegex } from "./util";
```

Comment:
It is just a bonus as `escapeRegExp` is *accidentally* being implemented in `util.ts`.